.ci/bump-release-version.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+checkForVariable() {
+  VAR_NAME=$1
+  if [ ! -v "$VAR_NAME" ]; then
+    echo "Error: Define $1 environment variable"
+    exit 1
+  fi
+}
+
+checkForVariable "READ_ONLY_TOKEN"
+
+CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
+git fetch --prune --unshallow
+NEW_RELEASE=$(git describe --always --abbrev=0 | cut -d '-' -f 2)
```

Comment:
this should be done in Github action, see comment on yml file

Reply:
Yes, but I have done this to bump in pom.xml file to use it in maven command

Reply:
Take this value from shell parameter that you already put in action

Reply:
Script should not do anything with git,.
It just does maven version bump, and deal with GitHub milestone

Reply:
It might be even make sense to make two shell scripts to do mvn update and GitHub update separately.

Yes make two scripts.

GitHub action should do:
1.Mvn version set
2.Commit+ push
3.Update GitHub milestone

Reply:
Created 2 shell scripts.